src/MVector.jl

```diff
@@ -99,3 +99,19 @@ macro MVector(ex)
         error("Use @MVector [a,b,c] or @MVector([a,b,c])")
     end
 end
+
+# Named field access for the first three elements, using the conventional field
+# names from 3D geometry
+function Base.getproperty(v::MVector, name::Symbol)
```

Comment:
Shouldn't this be `@inline` for safety? It would be quite bad for performance if `name` had to be checked at runtime.

Reply:
Sure. It'd really be too bad if performance randomly fell off a cliff because of this!

Another defensive alternative is to explicitly use `getfield` in the `getindex` implementation.  That will be kinder on the compiler anyway and it's quite easy, so I've done it as well.